news/13441.bugfix.rst

```diff
@@ -0,0 +1 @@
+Add the full file from a resume retry download to the cache.
```

Comment:
```suggestion
Resumed downloads are saved to the HTTP cache like any other normal download.
```

or "as appropriate".

We don't need to get into the implementation, all the user cares about is that resumed downloads are cached in the same way pip used to cache downloads.

Reply:
Applied